sections/header.liquid

```diff
@@ -458,18 +464,33 @@
       observer.observe(this.header);
     }
 
+    setPreventReveal() {
+      this.preventReveal = true;
+    }
+
     onScroll() {
       const scrollTop = window.pageYOffset || document.documentElement.scrollTop;
 
-      if (scrollTop > this.currentScrollTop && scrollTop > this.headerBounds.bottom) {
+      window.clearTimeout(this.isScrolling);
+
+      this.isScrolling = setTimeout(() => {
+        this.preventReveal = false;
+      }, 66);
```

Comment:
Is this needed when `if (this.preventReveal == false)`? If not, we can include it inside `if (this.preventReveal)` below.


Also, where is this `66` timeout coming from? 👀

Reply:
Done 👍  As for the `66` it was just something I used because of [the article reference](https://gomakethings.com/detecting-when-a-visitor-has-stopped-scrolling-with-vanilla-javascript/) I was using.